src/index.html

```diff
@@ -5,9 +5,34 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img src="images/imac.jpeg" alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+     class="card__image">
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__prodcode">Product code: 195434</p>
+      <p class="card__rating">
+        <span class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </span>
+              Reviews: 5
+      </p>
+      <p class="card__price">
+        <span class="card__price-title">Price:</span>
+        <span class="card__price-value">$2,199</span>
+      </p>
+      <a href="#" class="card__buy-button"
+      data-qa="hover">Buy </a>
```

Comment:
Remember to keep code style according to the [checklist (№14)](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)